### DISCUSSION TITLE
blog/what-is-survival-analysis/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#102] [General]
# blog/what-is-survival-analysis/

Learn more about survival analysis (also called time-to-event analysis), in which context and how it is used. Also learn how to apply it by hand and in R

https://statsandr.com/blog/what-is-survival-analysis/

**Comment by L-Groeninger** [comment]
Hey Antoine, this is great, thanks! Would have loved to read it when I did a course on survival analysis:) A couple of months ago, I also wrote sth. on reproducing an early influential paper on [survival analysis from scratch](https://lukas-r.blog/posts/2022-06-25-surviving-from-scratch/).

**Reply by AntoineSoetewey** [reply]
Thanks for your input Lukas!

I actually came across your post while I was writing mine, and I found it very thorough, clear, and accessible.
Thanks for sharing.

Regards,
Antoine

**Comment by irenerlover** [comment]
Your post is very informative, and the references attached in the end of the posts are very useful for the beginner. Appreciate your effort!

**Reply by AntoineSoetewey** [reply]
Glad you find it useful @doraemoniloveu!

If you are looking for a reference about more advanced survival models, I recommend this book:

- Legrand, C., 2021. *Advanced survival models.* CRC Press.

*Disclaimer: it is written by my PhD supervisor.*

**Comment by idumahg** [comment]
Short but very descriptive post. This really made me understand survival analysis as a novice. Thank you for your effort in putting this up.

**Reply by AntoineSoetewey** [reply]
Glad you find it useful @idumahg!

**Comment by ArthurPsyc** [comment]
This is an amazing resource! I hope I'm not too late to the party, but I wondered if you could speak to the appropriateness of using survival analysis when the event of interest is attrition/participant dropout itself (as well as identifying or associating predictor variables of attrition). It seems to make sense conceptually, but since I've not seen it done in these terms, I'm hesitant to move forward and having a difficult time finding a guide.

Any guidance you could offer would be greatly appreciated.

**Reply by AntoineSoetewey** [reply]
Dear @ArthurPsyc,

Thanks for your kind words!

Using survival analysis when the event of interest is attrition/dropout is indeed appropriate. There are actually many companies which use survival analysis techniques to summarize, infer and predict what is called "customer churn" (i.e., the fact that customers stop using your product or service).

If you are interested in this, you can:

- start with some Kaplan-Meier curves to have a better idea of when and by what extent your participants leave,
- do some log-rank tests to compare groups in terms of attrition,
- and if you would like to identify the predictor variables of attrition you can start with a Cox model (the most common regression model with survival data).

Note that some also use logistic regressions to identify variables associated with attrition, in case you are not interested in using methods from survival analysis. In this case, you are only interested in whether the participant leaves or not, but you do not consider the time until the event happens.

Hope this helps.

Regards,
Antoine

**Comment by ArthurPsyc** [comment]
Wow! This is a huge help, I really can't thank you enough. I really
appreciate the sources included in the article, and the time you spent
laying this out for me, and such a quick response.

This gives me some confidence to move forward with my thesis proposal. I'm
in a funny situation where most people I know have never even heard of
survival analyses (clinical psychology master's program), which is great in
terms of adding a novel idea to my project.. but. Tricky to find guidance.

Thank you again

All the best,
Arthur

**Reply by AntoineSoetewey** [reply]
You’re welcome!

Good luck for your thesis!

**Comment by lilyzhang4499-cmd** [comment]
I just started to learn about survival function. This article was extremely helpful and cleared up a lot of concepts. Thank you.

**Reply by AntoineSoetewey** [reply]
You're welcome @lilyzhang4499-cmd !
